Assets/HoloToolkit/Input/Scripts/Utilities/AttachToController.cs

```diff
@@ -11,20 +11,17 @@ namespace HoloToolkit.Unity.InputModule
     /// <summary>
     /// Waits for a controller to be instantiated, then attaches itself to a specified element
     /// </summary>
-    public class AttachToController : MonoBehaviour
+    public class AttachToController : ControllerFinder
     {
-#if  UNITY_WSA && UNITY_2017_2_OR_NEWER
-        public InteractionSourceHandedness Handedness { get { return handedness; } }
-
+    #if  UNITY_WSA && UNITY_2017_2_OR_NEWER
         [Header("AttachToController Elements")]
         [SerializeField]
-        protected InteractionSourceHandedness handedness = InteractionSourceHandedness.Left;
-
-#endif
+        protected new InteractionSourceHandedness handedness = InteractionSourceHandedness.Left;
+    #endif
         public MotionControllerInfo.ControllerElementEnum Element { get { return element; } }
```

Comment:
![image](https://user-images.githubusercontent.com/3580640/33789134-6b508956-dc2b-11e7-8e9a-e08460135e6c.png)

Looks like this was refactored out but not addressed here. It can probably be removed here, right?